packages/common/client-utils/src/bufferBrowser.ts

```diff
@@ -191,7 +192,9 @@ export class IsoBuffer extends Uint8Array {
 		let sanitizedStr = str;
 		// Remove everything after padding - Node buffer ignores everything
 		// after any padding whereas base64-js does not
-		sanitizedStr = sanitizedStr.split("=")[0];
+		const splitStr = sanitizedStr.split("=")[0];
+		assert(splitStr !== undefined, "splitStr is undefined in IsoBuffer.sanitizeBase64");
+		sanitizedStr = splitStr;
```

Comment:
This assert isn't logically needed - can we use a non-null assertion to avoid the extra check at runtime?
```suggestion
		// String split is guaranteed to return at least one member even if no splitting occurs (the original string).
		// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
		sanitizedStr = sanitizedStr.split("=")[0]!;
```